clang/include/clang/Basic/Attr.td

```diff
@@ -1060,11 +1060,6 @@ static llvm::StringRef canonicalizePlatformName(llvm::StringRef Platform) {
              .Case("ShaderModel", "shadermodel")
              .Default(Platform);
 }
-static llvm::StringRef getPrettyEnviromentName(llvm::Triple::EnvironmentType EnvironmentType) {
```

Comment:
This is no longer necessary, we can use `llvm::Triple::getEnvironmentTypeName()` instead.